test/evergreen.yml

```diff
@@ -4568,6 +4570,22 @@ tasks:
                 cat $t && rm $t
             fi
 
+            # Generate the code complexity statistics and upload the analysis into Atlas charts
+            virtualenv -p python3 venv
+            source venv/bin/activate
+            pip3 install pandas
```

Comment:
Suggest moving the 3 lines down (in front of line 4581) as that's where the pandas module is needed.